app/assets/stylesheets/application.scss

```diff
@@ -143,37 +175,33 @@ h6 {
 
 }
 
-.item-background-img {
-
-}
-
-#main {
+.main {
   position: relative;
   text-align: center;
+  margin-bottom: 35px
   h1 {
-    color: white;
+    color: #666666;
```

Comment:
Color literals like `#666666` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Unknown property margin-bottom-color